src/core/sys/posix/stdio.d

```diff
@@ -344,3 +344,7 @@ unittest
     assert(memcmp(ptr, testdata.ptr, testdata.length*wchar_t.sizeof) == 0);
     assert(fclose(f) == 0);
 }
+
+
+ssize_t getdelim (char** lineptr, size_t* n, int delimiter, FILE* stream);
+ssize_t getline (char** lineptr, size_t* n, FILE* stream);
```

Comment:
I obviously need to do a reading on DIP 1000. I've updated the PR, but this question is making me wonder:

```
void bar(scope char** c, scope int* n) @trusted
{

}

void foo() @safe
{
    char[100] buf;
    char* c = &buf[0];
    int n;

    bar(&c, &n);
}

void main()
{
    foo();
}
```

```
test.d(12): Error: cannot take address of scope local c in @safe function foo
```

Moving `char[100] buf` ouside of a scope works.